packages/server/src/schema/uuid/user/resolvers.ts

```diff
@@ -425,14 +426,18 @@ export const resolvers: Resolvers = {
       return { success: true, query: {} }
     },
 
-    async setDescription(_parent, { input }, { dataSources, userId }) {
+    async setDescription(_parent, { input }, context) {
+      const { userId, database } = context
       assertUserIsAuthenticated(userId)
-      const result = await dataSources.model.serlo.setDescription({
-        ...input,
+      if (input.description.length >= 64 * 1024) {
+        throw new UserInputError('description too long')
+      }
+      await database.mutate('update user set description = ? where id = ?', [
+        input.description,
         userId,
-      })
-
-      return { success: result.success, query: {} }
+      ])
+      await UuidResolver.removeCache({ id: userId }, context)
```

Comment:
```suggestion
      await dataSources.model.getUuid._querySpec.removeCache()
```

BTW this should work also so that this PR can be merged faster (than you can migrate the user query in a follow up PR)

Reply:
Hm, the test case still seems to fail in the same way.